botbot/templates/bots/request_success.html

```diff
@@ -1,6 +1,6 @@
 {% extends "landing/base.html" %}
 
 {% block content %}
-    <h2>Thank You!</h2>
-    <p>Your request has been sent off to the bots, we will be in touch soon.</p>
+    <h1>Thank You!</h1>
+    <p>Your request has been sent off to the bots. If your channel has been selected for logging we will contact you shortly.</p>
```

Comment:
Sure you want to use "we will contact you shortly"? Most of the questions we get are "when will you add my channel?"

Perhaps:

> Adding channels is a manual process and we prioritize logging large active channels. We do our best to get to new requests in a timely fashion, but we typically have a large backlog. It may be weeks before anyone follows up with you. Please be patient!